pom.xml

```diff
@@ -61,7 +61,6 @@
             <url>https://packages.confluent.io/maven/</url>
         </repository>
     </repositories>
-
```

Comment:
I see that you removed new line.
Please left it as it was before. Content file should be the same as in original branch.
Thank you.